.gitignore

```diff
@@ -4,4 +4,6 @@ dist/
 MANIFEST
 *.pyc
 venv/
+.venv/
 .idea/
+index.html
```

Comment:
I think we cannot have index.html here since this is committed in the gh-page branch. https://github.com/fedora-infra/tbs/tree/gh-pages

Reply:
Ok, then I will just add `.venv/`, so it corresponds with README